x-pack/plugins/ml/public/jobs/jobs_list/components/start_datafeed_modal/start_datafeed_modal.js

```diff
@@ -136,7 +138,21 @@ export class StartDatafeedModal extends Component {
           >
             <EuiModalHeader>
               <EuiModalHeaderTitle>
-                Start {(startableJobs.length > 1) ? `${startableJobs.length} jobs` : startableJobs[0].id}
+                <FormattedMessage
+                  id="xpack.ml.jobsList.startDatafeedModal.startJobsTitle"
+                  defaultMessage="Start {jobs}"
```

Comment:
Please look at my similar comment in `x-pack/plugins/ml/public/jobs/jobs_list/components/job_actions/results.js`.